docs/usage.md

```diff
@@ -1,331 +1,620 @@
+
+### Start with imputation `--steps impute`
+
+For starting from the imputation steps, the required flags are:
 
-1. The path must be enclosed in quotes
-2. The path must have at least one `*` wildcard character
-3. When using the pipeline with paired end data, the path must use `{1,2}` notation to specify read pairs.
+- `--steps impute`
+- `--input input.csv`: The samplesheet containing the input sample files in `bam`, `cram` or `vcf`, `bcf` format.
+- `--genome` or `--fasta`: The reference genome of the samples.
+- `--tools [glimpse1, quilt, stitch]`: A selection of one or more of the available imputation tools. Each imputation tool has their own set of specific flags and input files. These required files are produced by `--steps panelprep` and used as input in:
 
-If left unspecified, a default pattern is used: `data/*{1,2}.fastq.gz`
+  - `--chunks chunks.csv`: A samplesheet containing chunks per chromosome. These are produced by `--steps panelprep` using `GLIMPSE1`.
+  - `--posfile posfile.csv`: A samplesheet containing a `.legend.gz` file with the list of positions to genotype per chromosome. These are required by tools ( QUILT/STITCH/GLIMPSE1). It can also contain the `hap.gz` files (required by QUILT). The posfile can be generated with `--steps panelprep`.
+  - `--panel panel.csv`: A samplesheet containing the post-processed reference panel VCF (required by GLIMPSE1, GLIMPSE2). These files can be obtained with `--steps panelprep`.
 
-### `--single_end`
+#### Summary table of required parameters in `--steps impute`
 
-By default, the pipeline expects paired-end data. If you have single-end data, you need to specify `--single_end` on the command line when you launch the pipeline. A normal glob pattern, enclosed in quotation marks, can then be used for `--reads`. For example:
+|            | `--steps impute` | `--input` | `--genome` or `--fasta` | `--panel` | `--chunks` | `--posfile` |
+| ---------- | ---------------- | --------- | ----------------------- | --------- | ---------- | ----------- |
+| `GLIMPSE1` | ✅               | ✅ ¹      | ✅                      | ✅        | ✅         | ✅ ³        |
+| `GLIMPSE2` | ✅               | ✅ ¹      | ✅                      | ✅        | ✅         | ❌          |
+| `QUILT`    | ✅               | ✅ ²      | ✅                      | ❌        | ✅         | ✅ ⁴        |
+| `STITCH`   | ✅               | ✅ ²      | ✅                      | ❌        | ❌         | ✅ ³        |
+
+> ¹ Alignment files as well as variant calling format (i.e. BAM, CRAM, VCF or BCF)
+> ² Alignment files only (i.e. BAM or CRAM)
+> ³ `QUILT`: Should be a CSV with columns [panel id, chr, hap, legend]
+> ⁴ `GLIMPSE1 and STITCH`: Should be a CSV with columns [panel id, chr, legend]
+
+Here is a representation on how the input files will be processed depending on the input files type and the selected imputation tool.
+
+[![InputSoftwareCompatibility](images/InputSoftware_compatibility.png)](images/InputSoftware_compatibility.png)
```

Comment:
```suggestion
![InputSoftwareCompatibility](images/InputSoftware_compatibility.png)
```

Reply:
committing this to resolve a website issue (hopefully)

Reply:
Fixed in #153